providers/intercom/intercom.go

```diff
@@ -118,6 +120,10 @@ func userFromReader(reader io.Reader, user *goth.User) error {
 		return err
 	}
 
+	if u.EmailVerified != true {
+		return errors.New("User email not verified")
```

Comment:
I updated the PR.
Apps can use `user.RawData["email_verified"]` to get that value?